### PR TITLE
Improve UX of PlacePicker

### PR DESCRIPTION
**Pull request by jvaclavik**
Screenshot 2019-06-26 at 18 33 47

### CODE REVIEW

File: packages/universal-components/src/TagsInput/components/TagsContainer.js [hunk 1]
```diff
@@ -55,9 +55,8 @@ export default class TagsContainer extends React.Component<Props> {
 
 const styles = StyleSheet.create({
   tag: {
-    marginHorizontal: 3,
     alignSelf: 'center',
-    paddingHorizontal: 9,
-    paddingVertical: 6,
+    marginVertical: 2,
```
Comment (KarinaDavtyan) [inline packages/universal-components/src/TagsInput/components/TagsContainer.js]: ${defaultTokens.spaceXXXSmall}

File: packages/universal-components/src/TagsInput/components/TagsContainer.js [hunk 2]
```diff
@@ -55,9 +55,8 @@ export default class TagsContainer extends React.Component<Props> {
...
+    marginEnd: 4,
```
Comment (KarinaDavtyan) [inline packages/universal-components/src/TagsInput/components/TagsContainer.js]: ${defaultTokens.spaceXXSmall}

File: packages/universal-components/src/TagsInput/components/DeleteButton.js [hunk 3]
```diff
@@ -23,12 +25,25 @@ export default class DeleteButton extends React.PureComponent<Props> {
...
   }
 }
+
+const styles = StyleSheet.create({
+  buttonContainer: {
+    flexDirection: 'row',
+  },
+  text: {
+    color: defaultTokens.colorTextSecondary,
+    marginEnd: 2,
```
Comment (KarinaDavtyan) [inline packages/universal-components/src/TagsInput/components/DeleteButton.js]: ${defaultTokens.spaceXXXSmall}

File: packages/universal-components/src/TagsInput/TagsInput.js [hunk 4]
```diff
@@ -212,4 +243,17 @@ const styles = StyleSheet.create({
   opacityOne: {
     opacity: 1,
   },
+  row: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  inputFieldContainer: {
+    paddingVertical: 8,
```
Comment (KarinaDavtyan) [inline packages/universal-components/src/TagsInput/TagsInput.js]: ${defaultTokens.spaceXSmall}

File: apps/core/components/searchForm/PlacePicker/PlacePickerContent.js [hunk 5]
```diff
@@ -203,6 +203,19 @@ class PlacePickerContent extends React.Component<Props, State> {
     this.props.onClose();
   };
 
+  handleDeletePress = index => {
+    const locations = this.props[this.props.pickerType];
+
+    if (locations && locations.length !== 0) {
+      this.setState(state => ({
+        tempLocations: [
+          ...state.tempLocations.slice(0, index),
```
Comment (FilipMessa) [inline apps/core/components/searchForm/PlacePicker/PlacePickerContent.js]: what about use ramda for that?
https://ramdajs.com/docs/#remove

File: packages/universal-components/src/TagsInput/TagsInput.js [hunk 6]
```diff
@@ -212,4 +243,17 @@ const styles = StyleSheet.create({
...
+    height: 50,
+  },
+  inputFieldBorder: {
+    borderTopWidth: 1,
+    borderTopColor: '#eee',
```
Comment (FilipMessa) [inline packages/universal-components/src/TagsInput/TagsInput.js]: designTokens?

**Comment by FilipMessa** [comment]
just a few small notes, anyway LGTM

File: packages/universal-components/src/TagsInput/components/Tag.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  +fontSize: number,
+|};
+
+type TouchableTagIconProps = {|
+  +onPress?: number => void,
+  +children: React.Node,
+  +index: number,
+|};
+
+class TouchableTagIcon extends React.Component<TouchableTagIconProps> {
```
Comment (KarinaDavtyan) [inline packages/universal-components/src/TagsInput/components/Tag.js]: What about making in functional component?

File: packages/universal-components/src/TagsInput/components/DeleteButton.js (hunk 3 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/components/DeleteButton.js]: Done

File: packages/universal-components/src/TagsInput/TagsInput.js (hunk 4 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/TagsInput.js]: Done

File: packages/universal-components/src/TagsInput/components/TagsContainer.js (hunk 2 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/components/TagsContainer.js]: Done

File: packages/universal-components/src/TagsInput/components/TagsContainer.js (hunk 1 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/components/TagsContainer.js]: Done

File: packages/universal-components/src/TagsInput/TagsInput.js (hunk 6 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/TagsInput.js]: Done

File: apps/core/components/searchForm/PlacePicker/PlacePickerContent.js (hunk 5 above)
Comment (jvaclavik) [reply apps/core/components/searchForm/PlacePicker/PlacePickerContent.js]: Done

File: packages/universal-components/src/TagsInput/components/Tag.js (hunk 7 above)
Comment (jvaclavik) [reply packages/universal-components/src/TagsInput/components/Tag.js]: (eslint is complaining, we agreed that we will keep class component for now)

**Pull request merged**